src/test/java/it/cnr/isti/hpc/wikipedia/article/ArticleTest.java

```diff
@@ -58,4 +66,50 @@ protected static String readFileAsString(String filePath)
 		reader.close();
 		return fileData.toString();
 	}
+
+	@Test
+	public void testExtractingLinksOtherLang(){
+
+		String mediawiki = "* La Douceur de croire, pièce en trois actes, Théâtre Français, 8 July 1899\n" +
+				"\n" +
+				"[[:France]]In collaboration with [[:fr:André Delavigne]] " +
+				"* Blakson père [[::::::Potato|Pommes]] et fils,[[fr:Something]] comédie en quatre actes, Théâtre de l'Odéon\n" +
+				"* Les petites  marmites, comédie en trois actes, Théâtre du Gymnase\n" +
+				"[[cite:Gundam]] * Voilà Monsieur !, comédie en un acte, Théâtre du Gymnase";
+
+		ArticleParser parser = new ArticleParser(Language.EN);
+		Article a = new Article();
+		parser.parse(a, mediawiki);
+
+		Pair<List<String>, List<String>> anchorsAndUris = getAnchorsAndUris(a);
+		List<String> uris = anchorsAndUris.getSecond();
+
+		assertFalse(uris.contains("André_Delavigne"));
+		assertFalse(uris.contains("Something"));
+//		assertThat(uris, hasItems("France", "Potato", "cite:Gundam"));
+        assertThat(uris, hasItems("France", "Potato"));
+        assert(uris.size()==2);
```

Comment:
Mix of spaces and tabs looks odd

Reply:
Still mixed spaces and tabs, and I've just spotted that there should be spaces around the `==` too.